pyvast/setup.py

```diff
@@ -0,0 +1,37 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+from setuptools import setup
+
+with open("README.md", "r") as fh:
+    long_description = fh.read()
+
+setup(
+    author="Tenzir",
+    author_email="engineering@tenzir.com",
+    classifiers=[
+        # https://pypi.org/classifiers/
+        "Development Status :: 3 - Alpha",
+        "License :: OSI Approved :: BSD License",
+        "Operating System :: MacOS :: MacOS X",
+        "Operating System :: POSIX :: Linux",
```

Comment:
Should we add FreeBSD here?

Reply:
Unsure about such a bold claim. Since VAST is currently not tested on FreeBSD due to GitHub action runner limitations, I would prefer to not list it here.